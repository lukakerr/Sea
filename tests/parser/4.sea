fn main {} (
  num n = 5
  ret n * 2
)