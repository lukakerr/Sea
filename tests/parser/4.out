Main (Assignment (Equals "n" (Const (Number 5)) (Return (App (App (Prim Times) (Var "n")) (Const (Number 2))))))
